### ISSUE TITLE
Base:  Develop deploy and monitoring jobs for nodes

### ISSUE THREAD

**Issue by yillkid** [DLTcollab/general#141]
This issue depends on #123.

Discuss and develop all the deploy and monitoring jobs (Both of Ansible and Rundeck jobs) of IOTA full-node and Tangle-accelerator.

## Expected Outputs
- [ ] Deploy scripts
  - [ ] Start/restart/stop system service
  - [ ] Revise neighbor list
  - [ ] Revise load balancing backend
    - [ ] full-node
    - [ ] tangle accelerator

## Hook rules (WIP):

**Comment by ender503** [comment]
@yillkid Because the `Monitor exporters` is quite a large task, I move them into another issue #142. 😃

**Comment by yillkid** [comment]
The description of this issue could merge into #142, including:
1. Exporters rules
2. Webhook module
3. Ansible playbook task
